linera-core/src/client/mod.rs

```diff
@@ -2605,20 +2619,18 @@ where
                 application_permissions: application_permissions.clone(),
             };
             let operation = Operation::System(SystemOperation::OpenChain(config));
-            let certificate = match self.execute_block(vec![operation]).await? {
-                ExecuteBlockOutcome::Executed(certificate) => certificate,
-                ExecuteBlockOutcome::Conflict(_) => continue,
-                ExecuteBlockOutcome::WaitForTimeout(timeout) => {
-                    return Ok(ClientOutcome::WaitForTimeout(timeout));
-                }
-            };
+            let certificate: ConfirmedBlockCertificate =
```

Comment:
I guess this annotation can be removed again?

Reply:
Done https://github.com/linera-io/linera-protocol/pull/2832/commits/e2f4558e190047550dd9fdaf60b7162d9e7a5bc6